source/preprocessor.tex

```diff
@@ -752,6 +833,364 @@
 \end{codeblock}
 \end{example}
 
+\rSec1[cpp.embed]{Resource inclusion}
+\indextext{preprocessing directive!header inclusion}
+\indextext{preprocessing directive!source-file inclusion}
+\indextext{inclusion!source file|see{preprocessing directive, source-file inclusion}}%
+\indextext{\idxcode{\#include}}%
+
+\rSec2[cpp.embed.gen]{General}
+
+\pnum
+A preprocessing directive of the form
+\begin{ncsimplebnf}
+\terminal{\# embed <} h-char-sequence \terminal{>} \opt{pp-tokens} new-line
+\end{ncsimplebnf}
+searches a sequence of
+\impldef{sequence of places searched for an embedded resource}
+places for a resource identified uniquely by the specified sequence between
+the \tcode{<} and \tcode{>} delimiters.
+How the places are specified or the resource identified is
+\impldef{search locations for embedded resources specified with \tcode{<>}}.
+
+\pnum
+A preprocessing directive of the form
+\begin{ncsimplebnf}
+\terminal{\# embed "} q-char-sequence \terminal{"} \opt{pp-tokens} new-line
+\end{ncsimplebnf}
+searches for a resource identified by the specified sequence between the
+\tcode{"} delimiters.
+The named resource is searched for in an
+\impldef{manner of search for named resource}
+manner.
+If this search is not supported, or if the search fails, the directive is
+reprocessed as if it read
+\begin{ncsimplebnf}
+\terminal{\# embed <} h-char-sequence \terminal{>} \opt{pp-tokens} new-line
+\end{ncsimplebnf}
+with the identical contained sequence (including \tcode{>} characters, if any)
+from the original directive.
+
+\pnum
+\recommended A mechanism similar to, but distinct from, the
+\impldef{sequence of places searched for a header}
+search paths used for \tcode{\#include} \iref{cpp.include}
+is encouraged.
+
+\pnum
+Either form of the \tcode{\#embed} directive processes the
+\grammarterm{pp-tokens}, if present, just as in normal text.
+The \grammarterm{pp-tokens} shall then have the form
+\grammarterm{embed-parameter-seq}.
+
+\pnum
+A resource is a source of data accessible from the translation environment.
+A resource has an \gterm{implementation-resource-width}, which is the
+\impldef{size in bits of a resource}
+size in bits of the resource.
+If the \gterm{implementation-resource-width} is not an integral multiple of
+\libmacro{CHAR_BIT}, the program is ill-formed.
+Let \gterm{implementation-resource-count} be
+\gterm{implementation-resource-width} divided by \libmacro{CHAR_BIT}.
```

Comment:
We consider implementation-resource-width proper defined terms; please use \defn.

And remove the italics here, because that's not the defining appearance.